### ISSUE TITLE
Failed to run tests while building tarantool from source on CentOS 8

### ISSUE THREAD

**Issue by Vigor-jpg** [tarantool/tarantool#7044]
**Bug description**

I was build Tarantool from source and make test failed.

* OS: [Linux]
* OS Version: [CentOS 8.2]
* Architecture: [x86-64]
* Tarantool 2.10.0-beta2-286-gb494e762e

**Steps to reproduce**

I followed this instruction
```
dnf install -y epel-release
dnf install -y git gcc cmake3 libarchive make gcc-c++ zlib-devel \
readline-devel ncurses-devel openssl-devel libunwind-devel libicu-devel \
python3-pyyaml python3-six python3-gevent
git clone https://github.com/tarantool/tarantool.git --recursive
cd tarantool
git submodule update --init --recursive
make clean
rm CMakeCache.txt
mkdir build && cd build
cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo
make
make test
```

<details>
    <summary>MAKE TEST OUTPUT</summary>
    /root/tarantool/third_party/luajit/test/tarantool-tests/lj-601-fix-gc-finderrfunc.test.lua ................... ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/lj-49-bad-lightuserdata.test.lua ..................... ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/lj-426-arm64-incorrect-check-closed-uv.test.lua ...... ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/lj-584-bad-renames-for-sunk-values.test.lua .......... ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/gh-6371-string-char-no-arg.test.lua .................. ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/lj-494-table-chain-infinite-loop.test.lua ............ ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/gh-6098-fix-side-exit-patching-on-arm64.test.lua ..... ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/gh-4427-ffi-sandwich.test.lua ........................ ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/gh-4773-tonumber-fail-on-NUL-char.test.lua ........... ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/lj-430-maxirconst.test.lua ........................... ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/misclib-getmetrics-capi.test.lua ..................... ok    
/root/tarantool/third_party/luajit/test/tarantool-tests/lj-524-fold-conv-respect-src-irt.test.lua ............ ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/gh-6189-cur_L.test.lua ............................... ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/gh-6065-jit-library-smoke-tests.test.lua ............. ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/lj-375-ir-bufput-signed-char.test.lua ................ ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/or-232-unsink-64-kptr.test.lua ....................... ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/misclib-getmetrics-lapi.test.lua ..................... ok    
/root/tarantool/third_party/luajit/test/tarantool-tests/gh-6084-missed-carg1-in-bctsetr-fallback.test.lua .... ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/lj-flush-on-trace.test.lua ........................... ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/gh-4476-fix-string-find-recording.test.lua ........... ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/lj-671-arm64-assert-after-mremap.test.lua ............ ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/lj-505-fold-no-strref-for-ptrdiff.test.lua ........... ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/misclib-memprof-lapi.test.lua ........................ ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/gh-3196-incorrect-string-length.test.lua ............. ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/lj-695-ffi-vararg-call.test.lua ...................... ok   
/root/tarantool/third_party/luajit/test/tarantool-tests/gh-6227-bytecode-allocator-for-comparisons.test.lua .. ok   
All tests successful.
Files=26, Tests=63,  1 wallclock secs ( 0.13 usr  0.05 sys +  1.03 cusr  0.28 csys =  1.49 CPU)
Result: PASS
[ 98%] Built target tarantool-tests
Scanning dependencies of target LuaJIT-tests
Running LuaJIT-tests
[  1/502] lang/andor.lua --- smoke
[  2/502] lang/andor.lua --- allcases
[  3/502] lang/andor.lua --- tracefib
[  4/502] lang/assignment.lua --- local
[  5/502] lang/assignment.lua --- global
[  6/502] lang/assignment.lua --- local lhs in key on lhs
[  7/502] lang/assignment.lua --- global lhs in key on lhs
[  8/502] lang/compare.lua --- 1,2
[  9/502] lang/compare.lua --- 2,1
[ 10/502] lang/compare.lua --- 1,1
[ 11/502] lang/compare.lua --- 2
[ 12/502] lang/compare.lua --- 1
[ 13/502] lang/compare.lua --- 0
[ 14/502] lang/compare.lua --- pcall
[ 15/502] lang/compare.lua --- bit
[ 16/502] lang/compare.lua --- string 1 255
[ 17/502] lang/compare.lua --- String comparisons:
[ 18/502] lang/compare.lua --- obj_eq/ne
[ 19/502] lang/compare_nan.lua --- nan nan
[ 20/502] lang/compare_nan.lua --- nan one
[ 21/502] lang/compare_nan.lua --- one nan
[ 22/502] lang/compare_nan.lua --- nan 1
[ 23/502] lang/compare_nan.lua --- 1 nan
[ 24/502] lang/compare_nan.lua --- not nan nan
[ 25/502] lang/compare_nan.lua --- not nan one
[ 26/502] lang/compare_nan.lua --- not one nan
[ 27/502] lang/compare_nan.lua --- not nan 1
[ 28/502] lang/compare_nan.lua --- not 1 nan
[ 29/502] lang/constant/number.lua --- exp
[ 30/502] lang/constant/number.lua --- hex exp
[ 31/502] lang/constant/table.lua --- tnew
[ 32/502] lang/constant/table.lua --- tdup
[ 33/502] lang/for.lua --- direction
[ 34/502] lang/for.lua --- coerce to integer at 13
[ 35/502] lang/for.lua --- coerce to integer at 10
[ 36/502] lang/for.lua --- cannot coerce to integer at 10
[ 37/502] lang/length.lua --- length increasing and decreasing in loop
[ 38/502] lang/length.lua --- length increasing in loop with existing element
[ 39/502] lang/length.lua --- length decreasing in loop with erased element
[ 40/502] lang/modulo.lua --- integer equivalence
[ 41/502] lang/modulo.lua --- fractional equivalence
[ 42/502] lang/modulo.lua --- jit constant RHS
[ 43/502] lang/modulo.lua --- jit constant LHS, with exit
[ 44/502] lang/modulo.lua --- divide by zero
[ 45/502] lang/concat.lua --- Constant folding
[ 46/502] lang/concat.lua --- Fuse conversions to strings
[ 47/502] lang/concat.lua --- Fuse string construction
[ 48/502] lang/concat.lua --- CSE, sink
[ 49/502] lang/concat.lua --- CSE, two buffers in parallel, no sink
[ 50/502] lang/concat.lua --- Append, CSE
[ 51/502] lang/concat.lua --- Append, FOLD
[ 52/502] lang/concat.lua --- Append to buffer, sink
[ 53/502] lang/concat.lua --- Append to two buffers in parallel, no append, no sink
[ 54/502] lang/concat.lua --- Sink into side-exit
[ 55/502] lang/concat.lua --- Very long strings
[ 56/502] lang/self.lua --- trivial setget
[ 57/502] lang/table.lua --- tables as keys in tables
[ 58/502] lang/table.lua --- some tables as keys in tables
[ 59/502] lang/upvalue/closure.lua --- for
[ 60/502] lang/upvalue/closure.lua --- while
[ 61/502] lang/upvalue/closure.lua --- repeat
[ 62/502] lang/upvalue/closure.lua --- func
[ 63/502] lang/upvalue/closure.lua --- recursive type change
[ 64/502] lang/upvalue/closure.lua --- Don't mark upvalue as immutable if written to after prototype definition
[ 65/502] lang/coroutine.lua --- traceback
[ 66/502] lang/tail_recursion.lua --- self
[ 67/502] lang/tail_recursion.lua --- mutual
[ 68/502] lang/vararg_jit.lua --- 1
[ 69/502] lang/vararg_jit.lua --- 2
[ 70/502] lang/vararg_jit.lua --- 3
[ 71/502] lang/vararg_jit.lua --- 4
[ 72/502] lang/vararg_jit.lua --- 5
[ 73/502] lang/vararg_jit.lua --- 6
[ 74/502] lang/vararg_jit.lua --- 7
[ 75/502] lang/gc.lua --- rechain
[ 76/502] lang/gc.lua --- TSETM gc
[ 77/502] lang/goto.lua --- Basic goto and label semantics.
[ 78/502] lang/goto.lua --- Goto is not a keyword.
[ 79/502] lang/goto.lua --- Trailing label is considered to be out of scope.
[ 80/502] lang/goto.lua --- Simple loop with cross-jumping.
[ 81/502] lang/goto.lua --- Backwards goto must close upval.
[ 82/502] lang/goto.lua --- Break must close upval, even if closure is parsed after break.
[ 83/502] lang/goto.lua --- Label prevents joining to KNIL.
[ 84/502] lang/goto.lua --- Break resolved from the right scope.
[ 85/502] lang/goto.lua --- Do not join twice with UCLO.
[ 86/502] lang/meta/arith.lua --- op
[ 87/502] lang/meta/arith.lua --- lhs
[ 88/502] lang/meta/arith.lua --- rhs
[ 89/502] lang/meta/arith.lua --- meta only lhs
[ 90/502] lang/meta/arith.lua --- meta only rhs
[ 91/502] lang/meta/arith.lua --- defaults string, int
[ 92/502] lang/meta/arith.lua --- defaults int, string
[ 93/502] lang/meta/arith.lua --- defaults string, string
[ 94/502] lang/meta/arith.lua --- defaults string, kint
[ 95/502] lang/meta/arith.lua --- defaults kint, string
[ 96/502] lang/meta/arith_jit.lua --- assert rhs
[ 97/502] lang/meta/arith_jit.lua --- assert lhs
[ 98/502] lang/meta/arith_jit.lua --- assert both sides
[ 99/502] lang/meta/arith_jit.lua --- adjust no result to one result
[100/502] lang/meta/call.lua --- table
[101/502] lang/meta/call.lua --- userdata
[102/502] lang/meta/call.lua --- number
[103/502] lang/meta/call.lua --- table with changing metamethod
[104/502] lang/meta/call.lua --- jit table
[105/502] lang/meta/call.lua --- jit table rawget as metamethod
[106/502] lang/meta/call.lua --- jit number
[107/502] lang/meta/call.lua --- jit newindex pcall
[108/502] lang/meta/call.lua --- jit index pcall
[109/502] lang/meta/cat.lua --- default
[110/502] lang/meta/cat.lua --- lhs
[111/502] lang/meta/cat.lua --- rhs
[112/502] lang/meta/cat.lua --- mixed types
[113/502] lang/meta/cat.lua --- jit mixed types
[114/502] lang/meta/comp.lua --- __lt and __le xop
[115/502] lang/meta/comp.lua --- __lt and __le values
[116/502] lang/meta/comp_jit.lua --- coverage
[117/502] lang/meta/comp_jit.lua --- Mixed metamethods for ordered comparisons.
[118/502] lang/meta/eq.lua --- __eq xop
[119/502] lang/meta/eq.lua --- __eq values
[120/502] lang/meta/eq_jit.lua --- coverage
[121/502] lang/meta/eq_jit.lua --- non-constant objects
[122/502] lang/meta/framegap.lua --- untitled
[123/502] lang/meta/index.lua --- table 1
[124/502] lang/meta/index.lua --- table 2
[125/502] lang/meta/index.lua --- userdata
[126/502] lang/meta/index.lua --- string
[127/502] lang/meta/len.lua --- table
[128/502] lang/meta/len.lua --- userdata
[129/502] lang/meta/len.lua --- number
[130/502] lang/meta/newindex.lua --- table 1
[131/502] lang/meta/newindex.lua --- jit gaining href
[132/502] lang/meta/newindex.lua --- jit gaining aref
[133/502] lang/meta/newindex.lua --- resize
[134/502] lang/meta/newindex.lua --- str
[135/502] lang/meta/nomm.lua --- untitled
[136/502] lang/meta/debuginfo.lua --- table metamethods
[137/502] lang/meta/debuginfo.lua --- userdata metamethods
[138/502] lib/base/assert.lua --- pass through one
[139/502] lib/base/assert.lua --- pass through many
[140/502] lib/base/assert.lua --- raise on nil
[141/502] lib/base/assert.lua --- raise on false
[142/502] lib/base/error.lua --- no message
[143/502] lib/base/error.lua --- level 0
[144/502] lib/base/error.lua --- default level
[145/502] lib/base/error.lua --- default level in xpcall
[146/502] lib/base/error.lua --- level 2 in xpcall
[147/502] lib/base/getfenv.lua --- untitled
[148/502] lib/base/getsetmetatable.lua --- get __metatable
[149/502] lib/base/getsetmetatable.lua --- jit smoke
[150/502] lib/base/getsetmetatable.lua --- jit assorted
[151/502] lib/base/getsetmetatable.lua --- jit get primitive metatable
[152/502] lib/base/ipairs.lua --- small integer values
[153/502] lib/base/ipairs.lua --- jit key=value
[154/502] lib/base/ipairs.lua --- untitled
[155/502] lib/base/next.lua --- _G 1
[156/502] lib/base/next.lua --- as iterator
[157/502] lib/base/pairs.lua --- nometatable
[158/502] lib/base/pairs.lua --- empty metatable
[159/502] lib/base/pairs.lua --- _G
[160/502] lib/base/pairs.lua --- count
[161/502] lib/base/pcall_jit.lua --- square sum
[162/502] lib/base/pcall_jit.lua --- sqrt square sum
[163/502] lib/base/pcall_jit.lua --- sum with error
[164/502] lib/base/pcall_jit.lua --- sum or square
[165/502] lib/base/pcall_jit.lua --- sum or square with error
[166/502] lib/base/select.lua --- select #
[167/502] lib/base/select.lua --- select modf
[168/502] lib/base/select.lua --- select 1 
[169/502] lib/base/select.lua --- select 2
[170/502] lib/base/select.lua --- select vararg #
[171/502] lib/base/select.lua --- select vararg i
[172/502] lib/base/select.lua --- select vararg 4
[173/502] lib/base/select.lua --- varg-select specialisation requires guard against select
[174/502] lib/base/tonumber_tostring.lua --- tonumber int
[175/502] lib/base/tonumber_tostring.lua --- tonumber float
[176/502] lib/base/tonumber_tostring.lua --- tostring int / tonumber
[177/502] lib/base/tonumber_tostring.lua --- tostring float / tonumber
[178/502] lib/base/tonumber_tostring.lua --- tonumber table
[179/502] lib/base/tonumber_tostring.lua --- tostring int / tostring
[180/502] lib/base/tonumber_tostring.lua --- tostring table __tostring
[181/502] lib/base/tonumber_tostring.lua --- tostring table __tostring __call
[182/502] lib/base/tonumber_tostring.lua --- print calls overridden tostring
[183/502] lib/base/tonumber_tostring.lua --- tonumber base 2
[184/502] lib/base/tonumber_tostring.lua --- __tostring must be callable
[185/502] lib/bit.lua --- Minimal sanity checks.
[186/502] lib/bit.lua --- Basic argument processing.
[187/502] lib/bit.lua --- unop test vectors
[188/502] lib/bit.lua --- binop test vectors
[189/502] lib/bit.lua --- shift test vectors
[190/502] lib/bit.lua --- tohex test vectors
[191/502] lib/bit.lua --- Don't propagate TOBIT narrowing across two conversions.
[192/502] lib/math/abs.lua --- smoke
[193/502] lib/math/abs.lua --- argcheck
[194/502] lib/math/constants.lua --- pi
[195/502] lib/math/constants.lua --- huge
[196/502] lib/math/random.lua --- generally uniform in range [0, 1)
[197/502] lib/math/random.lua --- all in range [1, 10]
[198/502] lib/math/random.lua --- all in range [-3, 11]
[199/502] lib/string/metatable.lua --- __index metamethod is string library
[200/502] lib/string/byte.lua --- simple
[201/502] lib/string/byte.lua --- Fixed slice [i,i+k] or overflow
[202/502] lib/string/byte.lua --- Positive slice [i,len] or overflow
[203/502] lib/string/byte.lua --- Negative slice [-i,len] or underflow
[204/502] lib/string/byte.lua --- Positive slice [1,i] or overflow
[205/502] lib/string/byte.lua --- Negative slice [1,-i] or underflow
[206/502] lib/string/byte.lua --- Check for slot stack overflow
[207/502] lib/string/char.lua --- jit one char
[208/502] lib/string/char.lua --- jit until out of bounds
[209/502] lib/string/char.lua --- jit five chars
[210/502] lib/string/dump.lua --- Must unpatch modified bytecode with ILOOP/JLOOP etc.
[211/502] lib/string/dump.lua --- roundtrip constants
[212/502] lib/string/format/num.lua --- small denormals at low precision
[213/502] lib/string/format/num.lua --- smoke
[214/502] lib/string/format/num.lua --- easily enumerable cases of %a, %A
[215/502] lib/string/format/num.lua --- easily enumerable cases of %f
[216/502] lib/string/format/num.lua --- easily enumerable cases of %e
[217/502] lib/string/format/num.lua --- assorted
[218/502] lib/string/format/num.lua --- assorted %a
[219/502] lib/string/format/num.lua --- Cases where inprecision can easily affect rounding
[220/502] lib/string/format/num.lua --- ExploringBinary.com/print-precision-of-dyadic-fractions-varies-by-language/
[221/502] lib/string/format/num.lua --- ExploringBinary.com/incorrect-floating-point-to-decimal-conversions/
[222/502] lib/string/format/num.lua --- big f
[223/502] lib/string/format/num.lua --- RandomASCII.wordpress.com/2013/02/07/
[224/502] lib/string/len.lua --- smoke
[225/502] lib/string/len.lua --- argcheck
[226/502] lib/string/lower_upper.lua --- smoke
[227/502] lib/string/lower_upper.lua --- repeated
[228/502] lib/string/lower_upper.lua --- repeated with growing string
[229/502] lib/string/lower_upper.lua --- misc upper
[230/502] lib/string/lower_upper.lua --- misc lower
[231/502] lib/string/multiple_functions.lua --- string_op
[232/502] lib/string/rep.lua --- smoke
[233/502] lib/string/rep.lua --- versus concat
[234/502] lib/string/rep.lua --- misc
[235/502] lib/string/rep.lua --- separator
[236/502] lib/string/rep.lua --- iterate to table
[237/502] lib/string/rep.lua --- iterate to table with sep
[238/502] lib/string/rep.lua --- iterate and concat
[239/502] lib/string/reverse.lua --- misc
[240/502] lib/string/sub.lua --- smoke
[241/502] lib/string/sub.lua --- argcheck
[242/502] lib/string/sub.lua --- all bar substrings
[243/502] lib/string/sub.lua --- Positive slice [i,len] or overflow
[244/502] lib/string/sub.lua --- Negative slice [-i,len] or underflow
[245/502] lib/string/sub.lua --- Positive slice [1,i] or overflow
[246/502] lib/string/sub.lua --- Negative slice [1,-i] or underflow
[247/502] lib/string/sub.lua --- jit sub 1 eq
[248/502] lib/string/sub.lua --- jit sub 1 ne (contents)
[249/502] lib/string/sub.lua --- jit sub 1 ne (rhs too long)
[250/502] lib/string/sub.lua --- jit sub 1,2 ne
[251/502] lib/string/sub.lua --- jit sub 1,k eq
[252/502] lib/string/sub.lua --- jit sub 1,k ne (contents)
[253/502] lib/string/sub.lua --- jit sub 1,k ne (rhs too long)
[254/502] lib/string/sub.lua --- jit sub 1,2 eq
[255/502] lib/string/sub.lua --- jit sub 1,3 eq
[256/502] lib/string/sub.lua --- jit sub 1,4 eq
[257/502] lib/string/sub.lua --- jit sub i,i
[258/502] lib/table/concat.lua --- table.concat
[259/502] lib/table/concat.lua --- table.concat must inhibit CSE and DSE
[260/502] lib/table/concat.lua --- table.concat must inhibit CSE and DSE 2
[261/502] lib/table/concat.lua --- table.concat must inhibit CSE and DSE 3
[262/502] lib/table/insert.lua --- table.insert(t,i)
[263/502] lib/table/insert.lua --- table.insert(t,i,i)
[264/502] lib/table/new.lua --- table.new
[265/502] lib/table/remove.lua --- table.remove(t) removes correct entries
[266/502] lib/table/remove.lua --- table.remove(t) returns the removed entry
[267/502] lib/table/remove.lua --- table.remove(t, 1) removes and returns the first entry
[268/502] lib/table/remove.lua --- TSETR hash part
[269/502] lib/table/remove.lua --- TSETR write barrier
[270/502] lib/table/sort.lua --- numbers
[271/502] lib/table/sort.lua --- strings
[272/502] lib/table/sort.lua --- tables
[273/502] lib/coroutine/yield.lua --- Stack overflow on return (create)
[274/502] lib/coroutine/yield.lua --- Stack overflow on return (wrap)
[275/502] lib/coroutine/yield.lua --- cogen
[276/502] lib/coroutine/yield.lua --- cofunc
[277/502] lib/coroutine/yield.lua --- assorted
[278/502] lib/ffi/bit64.lua --- smoke tobit
[279/502] lib/ffi/bit64.lua --- smoke band
[280/502] lib/ffi/bit64.lua --- smoke shl
[281/502] lib/ffi/bit64.lua --- smoke tohex
[282/502] lib/ffi/bit64.lua --- tobit/band assorted C types
[283/502] lib/ffi/bit64.lua --- tobit/band negative unsigned enum
[284/502] lib/ffi/bit64.lua --- jit band/bor/bxor
[285/502] lib/ffi/bit64.lua --- jit shift/xor
[286/502] lib/ffi/bit64.lua --- jit rotate/xor
[287/502] lib/ffi/bit64.lua --- jit all ops
[288/502] lib/ffi/cdata_var.lua --- byte array allocations
[289/502] lib/ffi/cdata_var.lua --- int array allocations
[290/502] lib/ffi/copy_fill.lua --- misc
[291/502] lib/ffi/copy_fill.lua --- jit char[10]
[292/502] lib/ffi/copy_fill.lua --- jit char[1]
[293/502] lib/ffi/err.lua --- error in FFI metamethod: don't print metamethod frame.
[294/502] lib/ffi/err.lua --- tailcall in regular metamethod: keep metamethod frame.
[295/502] lib/ffi/err.lua --- error in FFI metamethod: set correct PC.
[296/502] lib/ffi/istype.lua --- 1
[297/502] lib/ffi/istype.lua --- 2
[298/502] lib/ffi/istype.lua --- 3
[299/502] lib/ffi/istype.lua --- 4
[300/502] lib/ffi/istype.lua --- 5
[301/502] lib/ffi/jit_array.lua --- smoke
[302/502] lib/ffi/jit_array.lua --- int array pointer arithmetic
[303/502] lib/ffi/jit_array.lua --- double array pointer arithmetic
[304/502] lib/ffi/jit_array.lua --- double array pointer comparisons
[305/502] lib/ffi/jit_array.lua --- constant offset in double array index
[306/502] lib/ffi/jit_array.lua --- fixed index of minus one
[307/502] lib/ffi/jit_array.lua --- uint8_t array element comparisons
[308/502] lib/ffi/jit_array.lua --- int32_t array bit/bswap tricks
[309/502] lib/ffi/jit_array.lua --- int32_t shift/rotate/and
[310/502] lib/ffi/jit_complex.lua --- field access
[311/502] lib/ffi/jit_complex.lua --- one element circular chain, named indexing
[312/502] lib/ffi/jit_complex.lua --- one element circular chain, array indexing
[313/502] lib/ffi/jit_complex.lua --- one-arg initialiser
[314/502] lib/ffi/jit_complex.lua --- two-arg initialiser
[315/502] lib/ffi/jit_complex.lua --- float<>double conversions
[316/502] lib/ffi/jit_complex.lua --- Complex struct field
[317/502] lib/ffi/jit_complex.lua --- Index overflow for complex is ignored
[318/502] lib/ffi/jit_misc.lua --- errno
[319/502] lib/ffi/jit_misc.lua --- string
[320/502] lib/ffi/jit_misc.lua --- fill
[321/502] lib/ffi/jit_misc.lua --- copy array elements
[322/502] lib/ffi/jit_misc.lua --- copy from string
[323/502] lib/ffi/jit_misc.lua --- copy structures
[324/502] lib/ffi/jit_misc.lua --- init struct from first field, complex
[325/502] lib/ffi/jit_misc.lua --- int array as parameterised type
[326/502] lib/ffi/jit_misc.lua --- double array as parameterised type
[327/502] lib/ffi/jit_misc.lua --- abi
[328/502] lib/ffi/jit_misc.lua --- typeof constructed typeof
[329/502] lib/ffi/jit_struct.lua --- iteration variable as field name
[330/502] lib/ffi/jit_struct.lua --- constant field names
[331/502] lib/ffi/jit_struct.lua --- constants from structure
[332/502] lib/ffi/jit_struct.lua --- adding to array elements
[333/502] lib/ffi/jit_struct.lua --- double indexing
[334/502] lib/ffi/jit_struct.lua --- structurally identical
[335/502] lib/ffi/jit_struct.lua --- structurally different
[336/502] lib/ffi/jit_struct.lua --- union
[337/502] lib/ffi/jit_struct.lua --- circular chain
[338/502] lib/ffi/jit_struct.lua --- int struct initialiser
[339/502] lib/ffi/jit_struct.lua --- double struct initialiser
[340/502] lib/ffi/jit_struct.lua --- pointer / int struct initialiser
[341/502] lib/ffi/jit_struct.lua --- unstable pointer/int type struct initialiser
[342/502] lib/ffi/jit_struct.lua --- upvalued int box
[343/502] lib/ffi/meta_tostring.lua --- tostring/typeof semi-roundtrip
[344/502] lib/ffi/meta_tostring.lua --- assorted union fields
[345/502] lib/ffi/redir.lua --- function
[346/502] lib/ffi/redir.lua --- variable
[347/502] lib/ffi/type_punning.lua --- float
[348/502] lib/ffi/type_punning.lua --- double
[349/502] lib/ffi/type_punning.lua --- i8
[350/502] lib/ffi/type_punning.lua --- p32/p64
[351/502] lib/ffi/type_punning.lua --- u16
[352/502] lib/ffi/type_punning.lua --- u16
[353/502] lib/ffi/type_punning.lua --- i16
[354/502] lib/ffi/type_punning.lua --- i16
[355/502] lib/ffi/type_punning.lua --- double
[356/502] lib/ffi/type_punning.lua --- double
[357/502] lib/ffi/type_punning.lua --- u32
[358/502] lib/ffi/type_punning.lua --- u64
[359/502] lib/contents.lua --- pre-5.2 base
[360/502] lib/contents.lua --- pre-5.2 base rawlen
[361/502] lib/contents.lua --- math
[362/502] lib/contents.lua --- pre-5.2 math
[363/502] lib/contents.lua --- pre-5.2 string
[364/502] lib/contents.lua --- pre-5.2 table.pack
[365/502] lib/contents.lua --- io
[366/502] lib/contents.lua --- io file
[367/502] lib/contents.lua --- pre-5.2 package
[368/502] lib/contents.lua --- bit
[369/502] lib/contents.lua --- ffi
[370/502] bc/constov.lua --- float
[371/502] bc/constov.lua --- int
[372/502] computations.lua --- ack
[373/502] computations.lua --- ack notail
[374/502] computations.lua --- fac
[375/502] computations.lua --- ffib
[376/502] computations.lua --- fib
[377/502] computations.lua --- nsieve
[378/502] computations.lua --- recsum
[379/502] computations.lua --- recsump
[380/502] computations.lua --- tak
[381/502] trace/exit_frame.lua --- global assignments
[382/502] trace/exit_frame.lua --- mutual recursion
[383/502] trace/exit_frame.lua --- multi-path mutual recursion
[384/502] trace/exit_frame.lua --- late mutual recursion
[385/502] trace/exit_growstack.lua --- Exit needs to grow stack before slot fill.
[386/502] trace/exit_growstack.lua --- Exit needs to grow stack after slot fill.
[387/502] trace/exit_jfuncf.lua --- everything
[388/502] trace/gc64_slot_revival.lua --- BC_KNIL
[389/502] trace/gc64_slot_revival.lua --- BC_VARG
[390/502] trace/phi/copyspill.lua --- mat4mul
[391/502] trace/phi/ref.lua --- rref points into invariant part 1
[392/502] trace/phi/ref.lua --- rref points into invariant part 2
[393/502] trace/phi/ref.lua --- rref points into invariant part 3
[394/502] trace/phi/ref.lua --- rref points into invariant part 4
[395/502] trace/phi/ref.lua --- rref points into invariant part 5
[396/502] trace/phi/ref.lua --- rref points into invariant part 6
[397/502] trace/phi/ref.lua --- variant slot, but no corresponding SLOAD i-1
[398/502] trace/phi/ref.lua --- variant slot, but no corresponding SLOAD i+1
[399/502] trace/phi/ref.lua --- variant slot, but no corresponding SLOAD side exit
[400/502] trace/phi/ref.lua --- dup lref from variant slot (suppressed)
[401/502] trace/phi/ref.lua --- const rref
[402/502] trace/phi/ref.lua --- dup rref (ok)
[403/502] trace/phi/ref.lua --- variant slot, no corresponding SLOAD
[404/502] trace/phi/ref.lua --- don't eliminate PHI if referenced from snapshot
[405/502] trace/phi/ref.lua --- don't eliminate PHI if referenced from snapshot
[406/502] trace/phi/ref.lua --- don't eliminate PHI if referenced from another non-redundant PHI
[407/502] trace/phi/rotate.lua --- rot8
[408/502] trace/phi/rotate.lua --- rot9
[409/502] trace/phi/rotate.lua --- rot18
[410/502] trace/phi/rotate.lua --- rotx
[411/502] trace/snap.lua --- gcexit
[412/502] trace/snap.lua --- top
[413/502] trace/snap.lua --- top2
[414/502] trace/stitch.lua --- octal
[415/502] trace/stitch.lua --- coroutines
[416/502] opt/dse/array.lua --- 1
[417/502] opt/dse/array.lua --- 2
[418/502] opt/dse/array.lua --- 3
[419/502] opt/dse/array.lua --- 4
[420/502] opt/dse/array.lua --- 5
[421/502] opt/dse/array.lua --- 6
[422/502] opt/dse/array.lua --- 7
[423/502] opt/dse/array.lua --- 8
[424/502] opt/dse/array.lua --- 9
[425/502] opt/dse/array.lua --- 10
[426/502] opt/dse/array.lua --- 11
[427/502] opt/dse/array.lua --- 12
[428/502] opt/dse/array.lua --- CALLL must inhibit DSE.
[429/502] opt/dse/field.lua --- 1. Store with same ref and same value. All stores in loop eliminated.
[430/502] opt/dse/field.lua --- 2. Store with different ref and same value. All stores in loop eliminated.
[431/502] opt/dse/field.lua --- 3. Store with different ref and different value. Cannot eliminate any stores.
[432/502] opt/dse/field.lua --- 4. Store with same ref and different value. 2nd store remains in loop.
[433/502] opt/dse/field.lua --- 5. Store with same ref, different value and aliased loads.
[434/502] opt/fold/kfold.lua --- operators
[435/502] opt/fold/kfold.lua --- abs
[436/502] opt/fold/kfold.lua --- atan2 ldexp
[437/502] opt/fold/kfold.lua --- minmax
[438/502] opt/fold/kfold.lua --- floorceil
[439/502] opt/fold/kfold.lua --- sqrt exp log trig
[440/502] opt/fold/kfold.lua --- exp
[441/502] opt/fwd/hrefk_rollback.lua --- https://github.com/LuaJIT/LuaJIT/issues/124
[442/502] opt/fwd/tnew_tdup.lua --- 1.
[443/502] opt/fwd/tnew_tdup.lua --- 2.
[444/502] opt/fwd/tnew_tdup.lua --- 3.
[445/502] opt/fwd/tnew_tdup.lua --- 4. HREFK not eliminated. Ditto for the EQ(FLOAD(t, #tab.hmask), k).
[446/502] opt/fwd/tnew_tdup.lua --- 5. HREFK not eliminated. Ditto for the EQ(FLOAD(t, #tab.hmask), k).
[447/502] opt/fwd/tnew_tdup.lua --- 6.
[448/502] opt/fwd/tnew_tdup.lua --- 7.
[449/502] opt/fwd/upval.lua --- 1. Open upvalue above base slot, aliasing an SSA value.
[450/502] opt/fwd/upval.lua --- 2. Open upvalue below base slot. UREFO CSE for a.x
[451/502] opt/fwd/upval.lua --- 3. Closed upvalue. UREFC CSE for a.x
[452/502] opt/fwd/upval.lua --- 4. Open upvalue below base slot. Forwarded. 1x USTORE (+ 1x DSE USTORE).
[453/502] opt/fwd/upval.lua --- 5. Closed upvalue. Forwarded. 1x USTORE (+ 1x DSE USTORE).
[454/502] opt/fuse.lua --- Don't fuse i+101 on x64.
[455/502] opt/loop/unroll.lua --- type instability on loop unroll
[456/502] opt/loop/unroll.lua --- untitled
[457/502] opt/loop/unroll.lua --- FAILFOLD on loop unroll
[458/502] opt/loop/unroll.lua --- Unroll if inner loop aborts.
[459/502] opt/sink/alloc.lua --- DCE or sink trivial TNEW or TDUP.
[460/502] opt/sink/alloc.lua --- Sink TNEW/TDUP
[461/502] opt/sink/alloc.lua --- Sink outermost table of nested TNEW.
[462/502] opt/sink/alloc.lua --- Sink one TNEW
[463/502] opt/sink/alloc.lua --- Sink TDUP or TDUP
[464/502] opt/sink/alloc.lua --- Sink of simplified complex add, unused in next iteration, drop PHI.
[465/502] opt/sink/alloc.lua --- Sink of complex add, unused in next iteration, drop PHI.
[466/502] opt/sink/alloc.lua --- Sink of TDUP with stored values that are both PHI and non-PHI.
[467/502] opt/sink/alloc.lua --- Sink of CONV.
[468/502] opt/sink/alloc.lua --- Sink of stores with numbers.
[469/502] opt/sink/alloc.lua --- Sink of stores with constants.
[470/502] opt/sink/alloc.lua --- Sink with two references to the same table.
[471/502] opt/sink/alloc.lua --- point
[472/502] opt/sink/alloc.lua --- untitled
[473/502] opt/sink/nosink.lua --- Cannot sink TNEW, aliased load.
[474/502] opt/sink/nosink.lua --- Cannot sink TNEW, escaping to upvalue.
[475/502] opt/sink/nosink.lua --- Cannot sink TNEW, escaping through a store.
[476/502] opt/sink/nosink.lua --- Cannot sink TNEW, iteratively escaping through a store.
[477/502] opt/sink/nosink.lua --- Cannot sink TNEW, escaping to next iteration (unused in 1st variant).
[478/502] opt/sink/nosink.lua --- Cannot sink TNEW, escaping to next iteration (snapshot ref).
[479/502] opt/sink/nosink.lua --- Cannot sink TNEW, escaping to next iteration (IR/snapshot ref).
[480/502] opt/sink/nosink.lua --- Cannot sink inner TNEW, escaping to next iteration (IR ref).
[481/502] opt/sink/nosink.lua --- Cannot sink TNEW, cross-PHI ref (and snapshot ref).
[482/502] opt/sink/nosink.lua --- Cannot sink TNEW, cross-PHI ref (and snapshot ref).
[483/502] opt/sink/nosink.lua --- Cannot sink TNEW, escaping to exit.
[484/502] opt/sink/nosink.lua --- Cannot sink TNEW, escaping to exit.
[485/502] opt/sink/nosink.lua --- Cannot sink, since nested inner table is non-PHI.
[486/502] opt/sink/ffi.lua --- incrementing
[487/502] opt/sink/ffi.lua --- hoistable increment
[488/502] opt/sink/ffi.lua --- escaping hoistable increment
[489/502] opt/sink/ffi.lua --- escaping addition
[490/502] opt/sink/ffi.lua --- conditional addition / incrementing
[491/502] opt/sink/ffi.lua --- incrementing pointer
[492/502] opt/sink/ffi.lua --- mutating complex
[493/502] opt/sink/ffi.lua --- mutating struct
[494/502] opt/sink/ffi.lua --- mutating struct 2
[495/502] opt/sink/ffi.lua --- escaping loop counter to float
[496/502] opt/sink/ffi.lua --- 64 bit crash bug
[497/502] opt/sink/ffi_nosink.lua --- escaping global
[498/502] opt/sink/ffi_nosink.lua --- preincrement escaping global
[499/502] opt/sink/ffi_nosink.lua --- escaping global and local
[500/502] opt/sink/ffi_nosink.lua --- swapping
[501/502] opt/sink/ffi_nosink.lua --- pointer to self
[502/502] opt/sink/ffi_nosink.lua --- strchr
502 passed
[ 98%] Built target LuaJIT-tests
Scanning dependencies of target lib21
[ 98%] Building C object third_party/luajit/test/PUC-Rio-Lua-5.1-tests/libs/CMakeFiles/lib21.dir/lib2.c.o
[ 98%] Linking C shared library lib21.so
[ 98%] Built target lib21
Scanning dependencies of target lib11
[ 98%] Building C object third_party/luajit/test/PUC-Rio-Lua-5.1-tests/libs/CMakeFiles/lib11.dir/lib1.c.o
[100%] Linking C shared library lib11.so
[100%] Built target lib11
Scanning dependencies of target lib1
[100%] Building C object third_party/luajit/test/PUC-Rio-Lua-5.1-tests/libs/CMakeFiles/lib1.dir/lib1.c.o
[100%] Linking C shared library lib1.so
[100%] Built target lib1
Scanning dependencies of target lib2
[100%] Building C object third_party/luajit/test/PUC-Rio-Lua-5.1-tests/libs/CMakeFiles/lib2.dir/lib2.c.o
[100%] Linking C shared library lib2.so
[100%] Built target lib2
Scanning dependencies of target PUC-Rio-Lua-5.1-tests-prepare
[100%] Copying lib2 to -lib2 for PUC-Rio Lua 5.1 tests
Create directory for PUC-Rio Lua 5.1 tests
[100%] Built target PUC-Rio-Lua-5.1-tests-prepare
Scanning dependencies of target PUC-Rio-Lua-5.1-tests
Running PUC-Rio Lua 5.1 tests
current path:
  ?
  /root/tarantool/third_party/luajit/test/PUC-Rio-Lua-5.1-tests/?.lua
  /root/.luarocks/share/lua/5.1/?.lua
  /root/.luarocks/share/lua/5.1/?/init.lua
  /root/.luarocks/share/lua/?.lua
  /root/.luarocks/share/lua/?/init.lua
  /usr/local/share/tarantool/?.lua
  /usr/local/share/tarantool/?/init.lua
  /usr/share/tarantool/?.lua
  /usr/share/tarantool/?/init.lua
  /usr/local/share/lua/5.1/?.lua
  /usr/local/share/lua/5.1/?/init.lua
  /usr/share/lua/5.1/?.lua
  /usr/share/lua/5.1/?/init.lua
  
  
    ---- total memory: 1K ----

testing lua.c options
/root/tarantool/build/src/tarantool -e "require[[strict]].off()"
Tarantool 2.10.0-beta2-286-gb494e762e
Target: Linux-x86_64-RelWithDebInfo
Build options: cmake . -DCMAKE_INSTALL_PREFIX=/usr/local -DENABLE_BACKTRACE=ON
Compiler: /usr/bin/cc /usr/bin/c++
C_FLAGS: -fexceptions -funwind-tables -fno-common -fopenmp -msse2 -std=c11 -Wall -Wextra -Wno-strict-aliasing -Wno-char-subscripts -Wno-format-truncation -Wno-gnu-alignof-expression -fno-gnu89-inline -Wno-cast-function-type
CXX_FLAGS: -fexceptions -funwind-tables -fno-common -fopenmp -msse2 -std=c++11 -Wall -Wextra -Wno-strict-aliasing -Wno-char-subscripts -Wno-format-truncation -Wno-invalid-offsetof -Wno-gnu-alignof-expression -Wno-cast-function-type

(the next error is expected by the test)
/root/tarantool/build/src/tarantool: option requires an argument -- 'e'

(the next error is expected by the test)
LuajitError: (command line):1: '=' expected near '<eof>'
fatal error, exiting the event loop

(the next error is expected by the test)
/root/tarantool/build/src/tarantool: invalid option -- 'f'
OK
testing garbage collection
.tables
strings
functions
..functions with errors
..long strings
............................weak tables
.........deep structures
....OK
    ---- total memory: 16K ----

testing debug library and debug information
+
....................................................................................................................+
OK
    ---- total memory: 1K ----

testing functions and calls
+
+
+
+
......................................................................................................................................................................................................................................................................OK
    ---- total memory: 1K ----

testing strings and string library
+
+
+
+
+
OK
    ---- total memory: 1K ----

testing scanner
+
+
+
+
+
OK
    ---- total memory: 1K ----

testing require
.+
testing assignments, logical operators, and constructors
+
OK
    ---- total memory: 1K ----

testing local variables plus some extra stuff
+
OK
    ---- total memory: 2K ----

testing syntax
+
+
.......+
........+
...........+
.......+
.OK
    ---- total memory: 2K ----

 >>> testC not active: skipping opcode tests <<<

testing string length overflow
+
.+
`querytab' nao ativo
`querytab' nao ativo
tables with table indices:
`querytab' nao ativo
tables with function indices:
`querytab' nao ativo
OK
    ---- total memory: 49K ----

testing tables, next, and for
+
+
....................................................................................................................+
+
.....+
.OK
    ---- total memory: 1K ----

testing pattern matching
+
+
+
+
OK
    ---- total memory: 2K ----

 >>> testC not active: skipping API tests <<<

    ---- total memory: 2K ----

testing metatables
......................................+
+
OK
    ---- total memory: 1K ----

testing vararg
+
+
+
OK
    ---- total memory: 1K ----

testing closures and coroutines
...+
.
 >>> testC not active: skipping yield/hook tests <<<

OK
    ---- total memory: 1K ----

testing errors
+
+
+
OK
    ---- total memory: 2K ----

testing numbers and math lib
+
`querytab' nao ativo
OK
    ---- total memory: 3K ----

testing sort
Sorting 30000 elements in 0.02 sec.
Re-sorting 30000 elements in 0.01 sec.
Invert-sorting other 30000 elements in 0.19 sec., with 473019 comparisons
Sorting 30000 equal elements in 0.10 sec.
............OK
    ---- total memory: 1K ----

testing large programs (>64k)
    ---- total memory: 1K ----

...`querytab' nao ativo
+
OK
.    ---- total memory: 2K ----

testing i/o
.+
..........................................................................................................................+
.+
+
+
..+
test done on 17/04/2022, at 16:00:57
Lua 5.1

tests not performed:
 >>> testC not active: skipping opcode tests <<<
 >>> testC not active: skipping API tests <<<
 >>> testC not active: skipping yield/hook tests <<<

final OK !!!
cleaning all!!!!
......    ---- total memory: 1K ----

total time: 4.74

[100%] Built target PUC-Rio-Lua-5.1-tests
Scanning dependencies of target lua-Harness-tests
Running lua-Harness tests
/root/tarantool/third_party/luajit/test/lua-Harness-tests/202-expr.t ......... ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/304-string.t ....... ok     
/root/tarantool/third_party/luajit/test/lua-Harness-tests/214-coroutine.t .... ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/101-boolean.t ...... ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/090-tap.t .......... ok   
/root/tarantool/third_party/luajit/test/lua-Harness-tests/091-profile.t ...... ok   
/root/tarantool/third_party/luajit/test/lua-Harness-tests/002-table.t ........ ok   
/root/tarantool/third_party/luajit/test/lua-Harness-tests/105-string.t ....... ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/308-io.t ........... 1/? LuajitError: stdin:1: '=' expected near 'U'
fatal error, exiting the event loop
/root/tarantool/third_party/luajit/test/lua-Harness-tests/308-io.t ........... ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/301-basic.t ........ ok     
/root/tarantool/third_party/luajit/test/lua-Harness-tests/241-standalone.t ... 1/? LuajitError: stdin:1: '=' expected near 'char(16)'
fatal error, exiting the event loop
/root/tarantool/third_party/luajit/test/lua-Harness-tests/241-standalone.t ... 2/? LuajitError: stdin:1: '=' expected near '\'
fatal error, exiting the event loop
LuajitError: stdin:2: '=' expected near '|'
fatal error, exiting the event loop
LuajitError: stdin:1: '=' expected near 'char(30)'
fatal error, exiting the event loop
/root/tarantool/third_party/luajit/test/lua-Harness-tests/241-standalone.t ... ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/203-lexico.t ....... ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/222-constructor.t .. ok     
/root/tarantool/third_party/luajit/test/lua-Harness-tests/221-table.t ........ ok     
/root/tarantool/third_party/luajit/test/lua-Harness-tests/309-os.t ........... sh: __IMPROBABLE__: command not found
/root/tarantool/third_party/luajit/test/lua-Harness-tests/309-os.t ........... ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/314-regex.t ........ ok       
/root/tarantool/third_party/luajit/test/lua-Harness-tests/015-forlist.t ...... ok     
/root/tarantool/third_party/luajit/test/lua-Harness-tests/000-sanity.t ....... ok   
/root/tarantool/third_party/luajit/test/lua-Harness-tests/307-math.t ......... ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/102-function.t ..... ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/310-debug.t ........ ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/403-jit.t .......... ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/402-ffi.t .......... ok     
/root/tarantool/third_party/luajit/test/lua-Harness-tests/303-package.t ...... ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/404-ext.t .......... ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/213-closure.t ...... ok     
/root/tarantool/third_party/luajit/test/lua-Harness-tests/014-fornum.t ....... ok     
/root/tarantool/third_party/luajit/test/lua-Harness-tests/212-function.t ..... ok     
/root/tarantool/third_party/luajit/test/lua-Harness-tests/200-examples.t ..... ok   
/root/tarantool/third_party/luajit/test/lua-Harness-tests/012-repeat.t ....... ok   
/root/tarantool/third_party/luajit/test/lua-Harness-tests/311-bit32.t ........ ok   
/root/tarantool/third_party/luajit/test/lua-Harness-tests/104-number.t ....... ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/401-bitop.t ........ ok     
/root/tarantool/third_party/luajit/test/lua-Harness-tests/108-userdata.t ..... ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/201-assign.t ....... ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/103-nil.t .......... ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/306-table.t ........ ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/320-stdin.t ........ 1/? LuajitError: stdin:1: '=' expected near 'char(19)'
fatal error, exiting the event loop
lua_debug> lua_debug> (debug command):1: dbg
/root/tarantool/third_party/luajit/test/lua-Harness-tests/320-stdin.t ........ ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/305-utf8.t ......... ok   
/root/tarantool/third_party/luajit/test/lua-Harness-tests/204-grammar.t ...... ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/231-metatable.t .... ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/411-luajit.t ....... skipped: only with LuaJIT
/root/tarantool/third_party/luajit/test/lua-Harness-tests/242-luac.t ......... skipped: LuaJIT
/root/tarantool/third_party/luajit/test/lua-Harness-tests/001-if.t ........... ok   
/root/tarantool/third_party/luajit/test/lua-Harness-tests/011-while.t ........ ok     
/root/tarantool/third_party/luajit/test/lua-Harness-tests/211-scope.t ........ ok     
/root/tarantool/third_party/luajit/test/lua-Harness-tests/107-thread.t ....... ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/223-iterator.t ..... ok   
/root/tarantool/third_party/luajit/test/lua-Harness-tests/106-table.t ........ ok    
/root/tarantool/third_party/luajit/test/lua-Harness-tests/232-object.t ....... ok     
All tests successful.
Files=50, Tests=1907,  4 wallclock secs ( 0.28 usr  0.11 sys +  2.78 cusr  0.83 csys =  4.00 CPU)
Result: PASS
[100%] Built target lua-Harness-tests
Scanning dependencies of target LuaJIT-test
[100%] Built target LuaJIT-test
Scanning dependencies of target test
Started ../../test/test-run.py --builddir=/root/tarantool/build --vardir=/root/tarantool/build/test/var
Running in parallel with 4 workers

Timeout options:
-------------------
SERVER_START_TIMEOUT:     90
REPLICATION_SYNC_TIMEOUT: 100
TEST_TIMEOUT:             110
NO_OUTPUT_TIMEOUT:        120

Collecting tests in 'app'            (Found 28  tests): application server tests.
Collecting tests in 'app-luatest'    (Found 7   tests): application server tests on luatest.
Collecting tests in 'app-tap'        (Found 50  tests): application server tests (TAP).
Collecting tests in 'box'            (Found 198 tests): Database tests.
Collecting tests in 'box-luatest'    (Found 26  tests): Database tests.
Collecting tests in 'box-py'         (Found 7   tests): legacy python tests.
Collecting tests in 'box-tap'        (Found 38  tests): Database tests with #! using TAP.
Collecting tests in 'engine'         (Found 85  tests): tarantool multiengine tests.
Collecting tests in 'engine-luatest' (Found 4   tests): Database tests.
Collecting tests in 'engine-tap'     (Found 4   tests): tarantool multiengine tap tests.
Collecting tests in 'engine_long'    (Found 4   tests): tarantool engine stress tests.
Collecting tests in 'long_run-py'    (Found 1   tests): long running tests.
Collecting tests in 'replication'    (Found 147 tests): tarantool/box, replication.
Collecting tests in 'replication-luatest' (Found 12  tests): replication luatests.
Collecting tests in 'replication-py' (Found 5   tests): tarantool/box, replication.
Collecting tests in 'sql'            (Found 124 tests): sql tests.
Collecting tests in 'sql-luatest'    (Found 4   tests): SQL tests on luatest.
Collecting tests in 'sql-tap'        (Found 544 tests): Database tests with #! using TAP.
Collecting tests in 'swim'           (Found 3   tests): SWIM tests.
Collecting tests in 'unit'           (Found 85  tests): unit tests.
Collecting tests in 'vinyl'          (Found 58  tests): vinyl integration tests.
Collecting tests in 'vinyl-luatest'  (Found 4   tests): vinyl space engine luatests.
Collecting tests in 'wal_off'        (Found 10  tests): tarantool/box, wal_mode = none.
Collecting tests in 'xlog'           (Found 15  tests): tarantool write ahead log tests.
Collecting tests in 'xlog-py'        (Found 6   tests): legacy python tests.

Tarantool server information
 | Found executable at /root/tarantool/build/src/tarantool
 | Found tarantoolctl at /root/tarantool/build/extra/dist/tarantoolctl

 | Tarantool 2.10.0-beta2-286-gb494e762e
 | Target: Linux-x86_64-RelWithDebInfo
 | Build options: cmake . -DCMAKE_INSTALL_PREFIX=/usr/local -DENABLE_BACKTRACE=ON
 | Compiler: /usr/bin/cc /usr/bin/c++
 | C_FLAGS: -fexceptions -funwind-tables -fno-common -fopenmp -msse2 -std=c11 -Wall -Wextra -Wno-strict-aliasing -Wno-char-subscripts -Wno-format-truncation -Wno-gnu-alignof-expression -fno-gnu89-inline -Wno-cast-function-type
 | CXX_FLAGS: -fexceptions -funwind-tables -fno-common -fopenmp -msse2 -std=c++11 -Wall -Wextra -Wno-strict-aliasing -Wno-char-subscripts -Wno-format-truncation -Wno-invalid-offsetof -Wno-gnu-alignof-expression -Wno-cast-function-type

======================================================================================
WORKR TEST                                            PARAMS          RESULT
---------------------------------------------------------------------------------
[003] xlog/snap_io_rate.test.lua                                      [ disabled ]
[001] xlog/panic_on_broken_lsn.test.lua                               [ disabled ]
[002] vinyl/gh-3395-read-prepared-uncommitted.test.l>                 [ disabled ]
[002] vinyl/errinj_tx.test.lua                                        [ disabled ]
[001] xlog/reader.test.lua                                            [ pass ]
[002] vinyl/throttle.test.lua                                         [ disabled ]
[001] xlog/errinj.test.lua                                            [ disabled ]
[004] xlog/header.test.lua                                            [ pass ]
[004] xlog/gh1433.test.lua                                            [ pass ]
[003] xlog/misc.test.lua                                              [ pass ]
[002] vinyl/tx_gap_lock.test.lua                                      [ pass ]
[003] xlog/gh-4771-upgrade.test.lua                                   [ pass ]
[002] vinyl/errinj_ddl.test.lua                                       [ disabled ]
[003] xlog/transaction.test.lua                                       [ pass ]
[003] xlog/big_tx.test.lua                                            [ pass ]
[004] xlog/gh-5894-pre-1.7.7-upgrade.test.lua                         [ pass ]
[003] xlog/checkpoint_threshold.test.lua                              [ disabled ]
[006] long_run-py/finalizers.test.py                                  [ disabled ]
[007] vinyl/select_consistency.test.lua                               [ disabled ]
[008] engine/conflict.test.lua                        memtx           [ pass ]
[001] xlog/force_recovery.test.lua                                    [ pass ]
[009] wal_off/iterator_lt_gt.test.lua                                 [ disabled ]
[002] vinyl/ddl.test.lua                                              [ pass ]
[002] vinyl/replica_rejoin.test.lua                                   [ disabled ]
[010] box-py/snapshot.test.py                                         [ pass ]
[010] box-py/iproto.test.py                                           [ pass ]
[007] vinyl/upsert.test.lua                                           [ pass ]
[008] engine/replica_join.test.lua                    vinyl           [ pass ]
[011] xlog/checkpoint_daemon.test.lua                                 [ pass ]
[012] app-luatest/http_client_test.lua                                [ pass ]
[013] box-py/call.test.py                                             [ pass ]
[013] box-py/print.test.py                                            [ pass ]
[011] xlog/panic_on_wal_error.test.lua                                [ pass ]
[014] app-luatest/gh_6817_console_memory_leak_test.l>                 [ pass ]
[008] engine/gh-4973-concurrent-alter-fails.test.lua  memtx           [ pass ]
[014] app-luatest/gh_6305_autocomplete_metamethod_te>                 [ pass ]
[014] app-luatest/msgpack_test.lua                                    [ pass ]
[013] box-py/args.test.py                                             [ pass ]
[014] app-luatest/gh_5747_crash_multiple_args_test.l>                 [ pass ]
[014] app-luatest/prbuf_test.lua                                      [ pass ]
[014] app-luatest/digest_crc32_recording_test.lua                     [ pass ]
[009] wal_off/alter.test.lua                                          [ pass ]
[015] app-tap/gh-4983-tnt-e-assert-false-hangs.test.>                 [ pass ]
[015] app-tap/tap.test.lua                                            [ pass ]
[016] swim/errinj.test.lua                                            [ disabled ]
[013] box-py/bootstrap.test.py                                        [ pass ]
[013] box-py/bad_trigger.test.py                                      [ pass ]
[016] swim/gh-5952-swim-serialize.test.lua                            [ pass ]
[015] app-tap/minimal.test.lua                                        [ pass ]
[015] app-tap/gh-4761-json-per-call-options.test.lua                  [ pass ]
[008] engine/tuple.test.lua                           memtx           [ pass ]
[015] app-tap/uri.test.lua                                            [ pass ]
[018] sql-tap/transitive1.test.lua                    memtx           [ pass ]
[008] engine/errinj_ddl.test.lua                      memtx           [ disabled ]
[018] sql-tap/collation.test.lua                      vinyl           [ pass ]
[019] wal_off/tuple.test.lua                                          [ pass ]
[018] sql-tap/count.test.lua                          vinyl           [ pass ]
[018] sql-tap/collation.test.lua                      memtx           [ pass ]
[008] engine/replica_join.test.lua                    memtx           [ pass ]
[019] wal_off/snapshot_stress.test.lua                                [ pass ]
[015] app-tap/gh-5632-6050-6259-gc-buf-reuse.test.lua                 [ pass ]
[015] app-tap/yaml.test.lua                                           [ pass ]
[015] app-tap/gh-5013-fiber-cancel.test.lua                           [ pass ]
[015] app-tap/logmod.test.lua                                         [ pass ]
[015] app-tap/table.test.lua                                          [ pass ]
[015] app-tap/gh-4491-coio-wait-leads-to-segfault.te>                 [ pass ]
[015] app-tap/errno.test.lua                                          [ pass ]
[015] app-tap/gh-5130-panic-on-invalid-log.test.lua                   [ pass ]
[015] app-tap/string.test.lua                                         [ pass ]
[015] app-tap/ssl-cert-paths-discover.test.lua                        [ pass ]
[015] app-tap/console.test.lua                                        [ pass ]
[015] app-tap/iconv.test.lua                                          [ fail ]
[015] Test failed! Output from reject file /root/tarantool/build/test/var/rejects/app-tap/iconv.reject:
[015] TAP version 13
[015] 1..11
[015] ok - check ascii->utf8 on simple string
[015] ok - check utf8->ascii on simple string
[015] ok - UTF conversion with ascii string
[015] ok - UTF conversion with non-ascii symbols
[015] 
[015] Last 15 lines of Tarantool Log file [Instance "app_server"][/root/tarantool/build/test/var/015_app-tap/iconv.test.lua.tarantool.log]:
[015] LuajitError: builtin/iconv.lua:85: iconv: Invalid argument
[015] fatal error, exiting the event loop
[Main process] Got failed test; gently terminate all workers...
[015] Worker "015_app-tap" got failed test; stopping the server...
[018] sql-tap/index4.test.lua                         vinyl           [ pass ]
[018] Worker "018_sql-tap" got signal to terminate; stopping the server...
[019] wal_off/func_max.test.lua                                       [ pass ]
[019] Worker "019_wal_off" got signal to terminate; stopping the server...
[008] engine/tuple.test.lua                           vinyl           [ pass ]
[008] Worker "008_engine" got signal to terminate; stopping the server...
---------------------------------------------------------------------------------
Occupied memory in failed tests (RSS, Mb):

Top 10 tests by occupied memory (RSS, Mb):
*  296.3 wal_off/alter.test.lua                                       
*  284.4 engine/tuple.test.lua:vinyl                                  
*  251.8 engine/tuple.test.lua:memtx                                  
*  113.7 sql-tap/index4.test.lua:vinyl                                
*   86.9 wal_off/func_max.test.lua                                    
*   80.5 vinyl/ddl.test.lua                                           
*   75.0 sql-tap/count.test.lua:vinyl                                 
*   73.9 engine/replica_join.test.lua:vinyl                           
*   70.8 engine/replica_join.test.lua:memtx                           
*   68.7 xlog/force_recovery.test.lua                                 

(Tests quicker than 0.1 seconds may be missed.)
---------------------------------------------------------------------------------
Duration of failed tests (seconds):
*   0.08 app-tap/iconv.test.lua                                       

Top 10 longest tests (seconds):
*  11.59 wal_off/alter.test.lua                                       
*   7.52 app-tap/gh-5632-6050-6259-gc-buf-reuse.test.lua              
*   5.79 vinyl/upsert.test.lua                                        
*   5.62 engine/replica_join.test.lua:vinyl                           
*   5.26 engine/tuple.test.lua:vinyl                                  
*   4.83 xlog/force_recovery.test.lua                                 
*   4.62 wal_off/func_max.test.lua                                    
*   4.53 engine/replica_join.test.lua:memtx                           
*   4.42 sql-tap/index4.test.lua:vinyl                                
*   4.19 vinyl/ddl.test.lua                                           
---------------------------------------------------------------------------------
Statistics:
* disabled: 14
* pass: 60
* fail: 1
Failed tasks:
- [app-tap/iconv.test.lua, null]
# logfile:        /root/tarantool/build/test/var/log/015_app-tap.log
# reproduce file: /root/tarantool/build/test/var/reproduce/015_app-tap.list.yaml
---
- [app-tap/gh-4983-tnt-e-assert-false-hangs.test.lua, null]
- [app-tap/tap.test.lua, null]
- [app-tap/minimal.test.lua, null]
- [app-tap/gh-4761-json-per-call-options.test.lua, null]
- [app-tap/uri.test.lua, null]
- [app-tap/gh-5632-6050-6259-gc-buf-reuse.test.lua, null]
- [app-tap/yaml.test.lua, null]
- [app-tap/gh-5013-fiber-cancel.test.lua, null]
- [app-tap/logmod.test.lua, null]
- [app-tap/table.test.lua, null]
- [app-tap/gh-4491-coio-wait-leads-to-segfault.test.lua, null]
- [app-tap/errno.test.lua, null]
- [app-tap/gh-5130-panic-on-invalid-log.test.lua, null]
- [app-tap/string.test.lua, null]
- [app-tap/ssl-cert-paths-discover.test.lua, null]
- [app-tap/console.test.lua, null]
- [app-tap/iconv.test.lua, null]
...
* undone: 1394
make[3]: *** [test/CMakeFiles/test.dir/build.make:77: test/CMakeFiles/test] Error 3
make[2]: *** [CMakeFiles/Makefile2:6391: test/CMakeFiles/test.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:6398: test/CMakeFiles/test.dir/rule] Error 2
make: *** [Makefile:2299: test] Error 2
</details>

**tarantool/build/test/var/015_app-tap/iconv.test.lua.tarantool.log**
```
LuajitError: builtin/iconv.lua:85: iconv: Invalid argument
fatal error, exiting the event loop
```
